File: C11/백준/Bronze/15969. 행복/행복.c
#include <stdio.h>

int main(void) {
	
	int N;
	scanf("%d", &N);
	
	int max = 0, min = 1000;
	
	for(int i = 0; i < N; i++) {
		int score;
		scanf("%d", &score);
		
		if(score > max) {
			max = score;
		}
		
		if(score < min) {
			min = score;
		}
	}
	
	printf("%d", max - min);
	
	return 0;
}

File: Java/백준/Bronze/15969. 행복/행복.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());

        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;

        String[] input = br.readLine().split(" ");

        for (int i = 0; i < N; i++) {
            int score = Integer.parseInt(input[i]);
            if (score > max) {
                max = score;
            }
            if (score < min) {
                min = score;
            }
        }

        System.out.print(max - min);
    }
}